flagsmith-engine/identities/traits/models.ts

```diff
@@ -1,8 +1,8 @@
 export class TraitModel {
-    traitKey: string;
+    traitKey: string | null;
```

Comment:
I don't think we should be allowing `traitKey` to be `null`, is there a reason this has been added?